addons/website/static/src/systray_items/relative_publish_time.js

```diff
@@ -0,0 +1,56 @@
+import { Component, onWillDestroy, xml } from "@odoo/owl";
+
+import { _t } from "@web/core/l10n/translation";
+
+const MINUTE = 60 * 1000;
+const HOUR = 60 * MINUTE;
+
+export class RelativePublishTime extends Component {
```

Comment:
See this comment: https://github.com/odoo/odoo/pull/156878#discussion_r1727323743

Reply:
Yes I had seen that. ;) I meant that instead of rewriting _everything_, you could just do `export class RelativePublishTime extends RelativeTime` (so the relation is clear), and then only overwrite `computeRelativeTime` without calling super.

And since you replace it in place, you don't need the `computeDeltaAndRelativeTime` hook.